src/libcore/iter/mod.rs

```diff
@@ -1031,17 +1031,15 @@ impl<I: ExactSizeIterator> ExactSizeIterator for Peekable<I> {}
 impl<I: Iterator> Peekable<I> {
     /// Returns a reference to the next() value without advancing the iterator.
     ///
-    /// The `peek()` method will return the value that a call to [`next()`] would
-    /// return, but does not advance the iterator. Like [`next()`], if there is
```

Comment:
It's all good. This was entirely me, not you. (Well, and partially github)


On Jul 6, 2016, 17:56 -0400, Tshepang Lekhonkhobe<notifications@github.com>, wrote:
>  
> Insrc/libcore/iter/mod.rs(https://github.com/rust-lang/rust/pull/33265#discussion_r69819098):
>  
> >@@ -1031,17 +1031,15 @@ impl<I: ExactSizeIterator>ExactSizeIterator for Peekable<I>{}>impl<I: Iterator>Peekable<I>{>/// Returns a reference to the next() value without advancing the iterator.>///>-  /// The `peek()` method will return the value that a call to [`next()`] would>-  /// return, but does not advance the iterator. Like [`next()`], if there is
>  
> I wish I could express meself better... I too often fail to find proper words to describe stuff.
>  
>  
> —
> You are receiving this because you were mentioned.
> Reply to this email directly,view it on GitHub(https://github.com/rust-lang/rust/pull/33265/files/e01e4cc4dbf657e411ecc16498e5f5c33d55f4d0#r69819098), ormute the thread(https://github.com/notifications/unsubscribe/AABsitkkqlKwXboovivMQ3Rvssi1aCKcks5qTCSZgaJpZM4ISXbk).
>  
>  
>